### PR TITLE
fix(list-header): RTL fix for list-header

### PR DESCRIPTION
**Pull request by sijav**
#### Short description of what this resolves:
RTL fix (scss changes) for list-header for wp, ios and android platform

#### Changes proposed in this pull request:

- Add RTL rule for list-header
**Ionic Version**: 2.x / 3.x

**Fixes**: #11211

### CODE REVIEW

**Comment by manucorporat** [comment]
Would you mind adding a before / after screenshot?

**Comment by AmitMY** [comment]
@manucorporat 
before
image


after
image

File: src/components/list/list.ios.scss [hunk 1]
```diff
@@ -182,6 +182,10 @@ $list-ios-header-background-color:    transparent !default;
   background: $list-ios-header-background-color;
 }
 
+[dir="rtl"] .list-header-ios {
+  padding-right: $list-ios-header-padding-left;
```
Comment (manucorporat) [inline src/components/list/list.ios.scss]: maybe renaming the css variable to `$list-ios-header-padding-start`?
thoughts @AmitMY @brandyscarney

Comment (AmitMY) [inline src/components/list/list.ios.scss]: @manucorporat 
@sijav and I had this discussion before.
Changing it to `start` will require support for `left`. which is possible and awesome, but should imo be a different PR, changing all relevant variables to `start`/`end` and adding the correct depredations, so in v4 can have a clean variables naming

Comment (manucorporat) [inline src/components/list/list.ios.scss]: @AmitMY  fair enough, LGTM

**Comment by manucorporat** [comment]
BTW, thanks everybody involved in these awesome RTL fixes. I am so happy we can finally support this, and 100% driven by the community!!

**Comment by brandyscarney** [comment]
Thank you again! Great work. :smile:
